addons/im_livechat/static/src/core/web/thread_actions.js

```diff
@@ -5,49 +5,27 @@ import { LivechatChannelInfoList } from "@im_livechat/core/web/livechat_channel_
 
 registerThreadAction("livechat-info", {
     actionPanelComponent: LivechatChannelInfoList,
-    condition(component) {
-        return (
-            component.thread?.channel_type === "livechat" &&
-            !component.isDiscussSidebarChannelActions
-        );
-    },
+    condition: ({ owner, thread }) =>
+        thread?.channel_type === "livechat" && !owner.isDiscussSidebarChannelActions,
     panelOuterClass: "o-livechat-ChannelInfoList bg-inherit",
     icon: "fa fa-fw fa-info",
-    iconLarge: "fa fa-fw fa-lg fa-info",
     name: _t("Information"),
     sequence: 10,
     sequenceGroup: 7,
     toggle: true,
 });
 registerThreadAction("livechat-status", {
     actionPanelComponent: LivechatChannelInfoList,
-    condition(component) {
-        return (
-            component.thread?.channel_type === "livechat" &&
-            !component.thread.livechat_end_dt &&
-            !component.isDiscussContent
-        );
-    },
-    dropdown: {
-        template: "im_livechat.LivechatStatusAction",
-        menuClass: "p-0",
-    },
+    condition: ({ owner, thread }) =>
+        thread?.channel_type === "livechat" && !thread.livechat_end_dt && !owner.isDiscussContent,
+    dropdown: 1,
```

Comment:
I'd rather having a boolean. Name doesn't suggest a boolean, I was wondering if it was a sequence, or something else?